### ISSUE TITLE
Doesn't work with Xubuntu boot CD

### ISSUE THREAD

**Issue by GammaRat** [m-khvoinitsky/dark-background-light-text-extension#37]
I've used this Add-on for some time.  I really like it.

However, after the update it has stopped working.  I even tested it with a Xubuntu boot CD.

The new update doesn't even default to a black background when enabled.

**Comment by m-khvoinitsky** [comment]
Please, provide more details. At least fIrefox version. Does it completely broken or partially? Screenshots?

**Comment by GammaRat** [comment]
The Xubuntu live CD is Xubuntu 15.10, which comes with FF 41.0.2.

No other  add-ons have been installed, and being a live CD it boots clean every time the PC is started.  

For some reason, the add-on doesn't appear to work at all.  For instance, after installation, you're typically automatically configured with the default profile.  So your screen inverts to white on black.   This isn't happening.  It's like the add-on isn't installed.  I can attempt any of the "profiles", such as "Simple CSS", and nothing changes on any of the webpages.  I've attempted to restart firefox to no effect.  I've attempted enabling and disabling to no effect.
